sql/src/main/java/io/crate/analyze/expressions/ExpressionAnalyzer.java

```diff
@@ -846,7 +847,7 @@ protected Symbol visitSubqueryExpression(SubqueryExpression node, ExpressionAnal
             /* note: This does not support analysis columns in the subquery which belong to the parent relation
              * this would require {@link StatementAnalysisContext#startRelation} to somehow inherit the parent context
              */
-            AnalyzedRelation relation = subQueryAnalyzer.analyze(node.getQuery());
+            QueriedRelation relation = subQueryAnalyzer.analyze(node.getQuery());
```

Comment:
Yep I changed the SelectSymbol constructor to require a QueriedRelation instead of an AnalyzedRelation